py/mphal.h

```diff
@@ -34,6 +34,9 @@
 #include <mphalport.h>
 #endif
 
+// For uintptr_t
+#include <stdint.h>
+
```

Comment:
This shouldn't be needed, I have fixed it in a different way in #5227 (define `mp_hal_stdio_poll` to nothing).

Reply:
OK, grabbed the same fix from that PR.